README.md

```diff
@@ -203,41 +203,41 @@ We use Docker in this project to assemble a different environment from different
 
 :point_right: _If you continue docker, it goes in the direction [Microservices](https://de.wikipedia.org/wiki/Microservices), Scaling and automation. We do not care, because we want to keep our Docker setup simple and use it only for local REDAXO development._
 
-### Was wird benötigt?
+### What is needed?
 
-Du musst nur [Docker (Community Edition) für dein System](https://www.docker.com/community-edition#/download) installieren, mehr wird nicht benötigt. In der Konfiguration musst du die Ordner freigeben, in denen Docker-Projekte laufen dürfen. Hier gibst du nun erstmal nur den Ordner an, in dem dieses Git-Repo liegt. Danach begibst du dich in deiner Konsole in diesen Ordner und startest die Container:
+You only have to [Docker (Community Edition) for your System](https://www.docker.com/community-edition#/download) install, more is not needed. In the configuration, you must share the folders where Docker projects are allowed to run. Here you give now only the folder, in which this Git repo lies. Then you go into this folder in your console and start the containers:
 
     $ docker-compose up -d
 
-Das wird beim ersten Mal ein kleines Weilchen dauern, weil zuerst die _Images_ runtergeladen werden müssen, aus denen Docker dann lauffähige Container baut. In deiner Konsole wird eine Menge Text vorbeilaufen.
+This will take a little while the first time, because first the _Images_ must be downloaded, from which Docker then builds working containers. There will be a lot of text in your console.
```

Comment:
This will take a little while the first time, because all the _Images_ must be downloaded, from which Docker will then build working containers. There will be a lot of text in your console.